doc/_posts/2014-03-11-capnproto-0.4.1-bugfixes.md

```diff
@@ -16,7 +16,7 @@ proceeding soon.
 
 In the meantime, though, there have been some major updates from the community:
 
-  * The folks at [CloudFlare](https://www.cloudflare.com/) have produced a
+  * The folks at [Cloudflare](https://www.cloudflare.com/) have produced a
```

Comment:
To be fair, at the time this blog post was published, CloudFlare was actually the preferred capitalization.

Reply:
Makes sense – I considered leaving existing posts as-is, but this way the old version is gone completely and won't show up anymore when grepping.